app/views/books/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= @book.title %>
+<% @book.reviews.each do |review| %>
+  <%= review.body %>
+  <%= review.rating %>
+  <%= review.user.name %>
+<% end %>
+
+<h1>Highest Rating: <%= @max %></h1>
+  <% @max_review = Review.find_by(rating: @max) %>
```

Comment:
You've already noted that you know this should live in a model.  One other thing to think about is that in order to accomplish this you have to make two queries. One to find @max and then one to find the Review.  I would probably lean towards building a method on your instance of a book that uses ActiveRecord to find out which review has the highest rating.